recipes/pcaone/meta.yaml

```diff
@@ -11,28 +11,31 @@ source:
   sha256: '{{ sha256 }}'
 
 build:
-  number: 0
-  skip: True  # [win]
+  number: 1
 
 requirements:
   build:
-    - make
     - {{ compiler('c') }}
     - {{ compiler('cxx') }}
+    - make
+    - intel-openmp
 
   host:
-    - mkl
+    - intel-openmp
+    - mkl >=2020.4
     - mkl-include
     - zlib
 
   run:
-    - mkl
-    - mkl-include
+    - intel-openmp
+    - mkl >=2020.4
     - zlib
 
 test:
   commands:
     - PCAone 
+    - ldd `which PCAone.avx2`  # [linux]
```

Comment:
Do you want to check something specific here? e.g. grepping for something in the output of ldd?

Reply:
I am testing something here. turns out ```intel-openmp``` and ```llvm-openmp``` can't coexist. I'll colse this.